CondTools/DT/test/validate/DTMtimeValidateDBRead.cc

```diff
@@ -53,27 +53,50 @@ void DTMtimeValidateDBRead::analyze(const edm::Event& e, const edm::EventSetup&
   float mTrms;
   float ckmt;
   float ckrms;
-  DTMtime::const_iterator iter = mT->begin();
-  DTMtime::const_iterator iend = mT->end();
-  while (iter != iend) {
-    const DTMtimeId& mTId = iter->first;
-    const DTMtimeData& mTData = iter->second;
-    status = mT->get(mTId.wheelId, mTId.stationId, mTId.sectorId, mTId.slId, mTime, mTrms, DTTimeUnits::counts);
-    if (status)
-      logFile << "ERROR while getting sl Mtime " << mTId.wheelId << " " << mTId.stationId << " " << mTId.sectorId << " "
-              << mTId.slId << " , status = " << status << std::endl;
-    if ((fabs(mTData.mTime - mTime) > 0.01) || (fabs(mTData.mTrms - mTrms) > 0.0001))
-      logFile << "MISMATCH WHEN READING sl Mtime " << mTId.wheelId << " " << mTId.stationId << " " << mTId.sectorId
-              << " " << mTId.slId << " : " << mTData.mTime << " " << mTData.mTrms << " -> " << mTime << " " << mTrms
-              << std::endl;
-    iter++;
-  }
+  if (readLegacyVDriftDB) {  //legacy format
+    edm::ESHandle<DTMtime> mT;
+    context.get<DTMtimeRcd>().get(mT);
+    std::cout << mT->version() << std::endl;
+    std::cout << std::distance(mT->begin(), mT->end()) << " data in the container" << std::endl;
```

Comment:
My understanding is that the use of std::cout is discouraged. Please use the relevant MessageLogger instead, I see that you did that in different parts of the PR

Reply:
@tvami, this is a code in the test area. Usually there are lighter requirements for code in \test, which is typically used in standalone user runs

Reply:
Indeed this appears to be a standalone test which is expected to produce, among other things, some formatted text output (cf the several couts in the file) - ie the output is part of the purpose of the test, therefore I'd prefer not to mess with it given that this is not the purpose of this PR. 
Those I modifed elsewhere are essentiallly warnings so I think it makes more sense and is safer to have them fixed.

Reply:
Thanks, it's good to know that \test has lighter requirements!